tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -217,7 +217,6 @@ def test_pantsd_aligned_output(self):
 
     for cmd, run in zip(cmds, daemon_runs):
       stderr_output = run.stderr_data.strip()
-      self.assertEqual(stderr_output, '', 'Non-empty stderr for {}: {}'.format(cmd, stderr_output))
```

Comment:
The issue I'm fixing here: Test expects stderr to be empty. However, I do not believe that assertion is always true. If one was to log, stderr would have the log output in stderr. 

Test is currently failing in CI because `INFO] waiting for file /home/travis/build/pantsbuild/pants/.pants.d/tmp/tmptb5bb5j_.pants.d/.pids/pantsd/pid to appear...` is present in stderr. I do not believe that is an issue.

Reply:
Mm. That would be because pantsd is taking longer to start than it did before. I don't think it is a problem though... it's entirely possible that it is environmental.

`stderr_output` is now unused though, and I would expect that to result in a lint error.

Reply:
@stuhood should we then bump the timeout for this pantsd code? Do you know @OniOni if that would fix this?

I’ll wait to merge until confirming this.

Reply:
So fwiw, I can only repro this "error" in CI. So I don't think we need to bump the timeout. (Also we're hitting the soft deadline of 5 secs, not the 60 hard deadline).

As for writing to `stderr`, I was able to confirm that we indeed log to `stderr` in `process_manager.py`.

```
 PANTS_CONFIG_FILES="$(pwd)/pants.daemon.ini" ./pants3 goals > /dev/null 2> stderr.txt && wc stderr.txt
       2       2      12 stderr.txt
```

I added a log line to process manager, so I don't have to hit soft deadline.
```diff
diff --git a/src/python/pants/pantsd/process_manager.py b/src/python/pants/pantsd/process_manager.py
index e42cbfa6e..666229dab 100644
--- a/src/python/pants/pantsd/process_manager.py
+++ b/src/python/pants/pantsd/process_manager.py
@@ -126,6 +126,9 @@ class ProcessMetadataManager(object):
     now = time.time()
     deadline = now + timeout
     info_deadline = now + info_interval
+
+    logger.info('test')
+
     while 1:
       if closure():
         return True
```

Sounds like that should not be the case @stuhood? Happy to add fix if that is the case.

Reply:
> As for writing to stderr, I was able to confirm that we indeed log to stderr in process_manager.py.

Yea, I was just talking about the unused variable (now removed).

This is all fine from my perspective. Will land this.

Reply:
Thanks, misread your comment 🤦‍♂️